components/mixins/experience-event/industry-verticals/experienceevent-reservation-details.schema.json

```diff
@@ -0,0 +1,146 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experience-event/industry-verticals/experienceevent-reservation-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Reservation Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["travel_and_hospitality"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Reservation details such as length, modification, refundable status, and number of rooms.",
+  "definitions": {
+    "reservation-details": {
+      "properties": {
+        "xdm:reservations": {
```

Comment:
Can we combine "search" and "hotel-flight-bookings" under the same object "reservation-details" ?